### DISCUSSION TITLE
Eufy Motion Sensor not detecting motion

### DISCUSSION THREAD

**Discussion by jamietomo** [fuatakgun/eufy_security#980] [Q&A]
Everything is working perfectly except for my motion sensors that aren't detecting any motion activity in Home Assistant. Is there anyway to resolve this?

**Comment by fuatakgun** [comment]
Be sure that you have enabled all push notifications from app so add-on can receive almost instant updates from devices

**Reply by V4ler1an** [reply]
fuatakgun, I too have an issue with this, there's a couple of things I don't understand though. With the Eufy motion sensors I have to have push notfications on for it to work with HA. Yet my Eufy door sensors I don't? I can turn their push notifications off and everything still works. Why the difference? My second question is to do with Eufy Security app user accounts. When I setup the integration with HA I specifically created a second user dedicated to HA with the belief that this would allow me to turn on all the annoying notifications for that user while keeping them off for my personal user. However, it seems any settings I change are retained for both users. Do I have to have a second phone running just so I can log it on with HA's user and turn on notifications?

**Reply by fuatakgun** [reply]
Push notifications are enabled on a device basis, so it is how eufy designed it.

Door sensors might be updating their status into cloud and integration might be picking them up but I am really not sure as I don't have any door/motion sensor.

**Reply by V4ler1an** [reply]
Ok, any possibility of testing the door sensor (they're cheap at the moment) to see if that method can be copied for the motion sensors? The motion sensors are by far the biggest annoyance because they trigger so very often about the house and the notifications are 99% worthless spam when you're at home. It's a relief that I can disable the door sensors but the motion notifictions are something of a nightmare.

**Reply by V4ler1an** [reply]
Update: it turn out that since adding my Eufy motion sensors a new Eufy app notification category of 'Eufy motion' has been added. This looks to only apply to the Eufy motion sensors so problem solved!

**Reply by jamietomo** [reply]
Where is this Eufy Motion category? I have the same issue.

**Reply by V4ler1an** [reply]
Well on Android 13 (OnePlus) it's: Settings > Apps > App management > Eufy Security > Manage notifications (scroll down to Categories).

**Reply by jamietomo** [reply]
Ah I'm on an iPhone. I can get it working by enabling notifications in the Eufy app but then it beeps at me every time I move which isn't ideal. If I just able 'React to motion' in the app then the Eufy app shows the motion detections (without the annoying notifications) but HA won't then show them.

**Reply by V4ler1an** [reply]
Yep that was exactly my experience. Does iOS not have notification categories? Surely there is some equivalent?

**Reply by fuatakgun** [reply]
No, there is no setting for ios

**Reply by t9001** [reply]
Wondering if anyone found a solution to this. My door and motion sensors both need notifications to be enabled for Home Assistant to receive their activity. Otherwise the door sensor in HA just stays "closed" and the motion sensor stays "clear". No way I can have notifications on in the Eufy app for these devices in "Home" mode, my phone would explode. But I would like HA to keep track of motion and door openings for automation purposes.
